components/thumbnail/stories/template.js

```diff
@@ -125,3 +126,38 @@ export const Template = ({
 		</div>
 	`;
 };
+
+export const SizingGroup = (args, context) => {
```

Comment:
This is taken directly from the Avatar template for now, but i'm eyeing @jawinn's work in #2883 as a better option